src/modules/sanctions/server/functions/banPlayer.lua

```diff
@@ -0,0 +1,34 @@
+---banPlayer
+---@param license string
+---@param reason string
+---@param time table
+---@return void
+function MOD_Sanctions:banPlayer(license, reason, time)
+    if (not license or not reason or not time) then
+        return
+    end
+
+    if (MOD_Sanctions:isPlayerBan(license)) then
+        NCS:coreTrace(("Player %s is already banned."):format(license))
+        return
+    end
+
+    time = API_Maths:unformatTime(time)
+    time = API_Maths:addTimeToUnix(time)
+    local ban_id <const> = ("BAN-%s-%s"):format(API_Strings:randomString(3), API_Strings:randomString(5))
+
+    local query <const> = ("INSERT INTO ncs_bans (ban_id, license, reason, time) VALUES ('%s', '%s', '%s', '%i')"):format(ban_id, license, reason, time)
+    API_Database:query(query, {}, function(result)
+        if (not result) then
+            NCS:coreTrace(("Failed to insert ban for player %s."):format(license))
+            return
+        end
```

Comment:
@CsokiHUN What do you think?

Reply:
its good, but why not store all identifiers?

Reply:
> its good, but why not store all identifiers?

bc the license was the only one identifier can't change ( if he changed rockstar account it's change btw )

Reply:
> its good, but why not store all identifiers?

It's not that, the "return" leaves me confused